src/deepsparse/loggers/constants.py

```diff
@@ -23,11 +23,13 @@
     "validate_identifier",
     "REQUEST_DETAILS_IDENTIFIER_PREFIX",
     "RESOURCE_UTILIZATION_IDENTIFIER_PREFIX",
+    "PREDEFINED_IDENTIFIER_ALIAS",
 ]
 
 UNSUPPORTED_IDENTIFIER_CHARS = {".", "[", "]"}
 REQUEST_DETAILS_IDENTIFIER_PREFIX = "request_details"
 RESOURCE_UTILIZATION_IDENTIFIER_PREFIX = "resource_utilization"
+PREDEFINED_IDENTIFIER_ALIAS = "predefined"
```

Comment:
based on usage, looks like this might be cleaner unless we add more helper fns
```suggestion
PREDEFINED_IDENTIFIER_ALIAS = "_predefined"
```

Reply:
Are you sure?
`"image_classification_pipeline/predefined".endswith("predefined") == True`
`"image_classification_pipeline/predefined".endswith("predefined") == False`

Reply:
Given that we keep the convention of the config file:
```yaml
loggers:
    list_logger:
        path: tests/deepsparse/loggers/helpers.py:ListLogger
data_logging:
    predefined: # `_predefined` is ugly imo
    - func: image_classification
      frequency: 1"""
```